packages/web-components/src/components/footer/footer-composite.ts

```diff
@@ -244,14 +249,10 @@ class DDSFooterComposite extends ModalRenderMixin(HybridRenderMixin(HostListener
       _languageCallback: languageCallback,
     } = this;
 
-    console.log(languageSelectorLabel);
+    const parsedList = !langList ? undefined : JSON.parse(langList as string);
```

Comment:
You can use `lit-html` property binding syntax to avoid having to de-serialize string: https://lit-html.polymer-project.org/guide/writing-templates#bind-to-properties